bin/cli.js

```diff
@@ -14,9 +14,18 @@ if (argv._.length < 3) {
   process.exit(1);
 }
 
+// Parse from/to as either a Regex or a string
+const isRegexMatch = /.*\/([gimy]*)$/;
+const getRegexOrString = value => {
+  if(!isRegexMatch.test(value)) return value;
+  const flags = value.replace(/.*\/([gimy]*)$/, '$1');
+  const pattern = value.replace(new RegExp('^/(.*?)/'+flags+'$'), '$1');
```

Comment:
This will not pass the linter, could you run ESLint on the code to fix style inconsistencies please?